resolver/caching_resolver.go

```diff
@@ -249,16 +250,16 @@ func isResponseCacheable(msg *dns.Msg) bool {
 	return !msg.Truncated && !msg.CheckingDisabled
 }
 
-func (r *CachingResolver) putInCache(cacheKey string, response *model.Response, ttl time.Duration,
-	publish bool,
+func (r *CachingResolver) putInCache(
+	ctx context.Context, cacheKey string, response *model.Response, ttl time.Duration, publish bool,
```

Comment:
This isn't super relevant to the functionality of the pull request, but is there a preference within the project to keep these parameters on the same line vs listing them one per line when the line length becomes too long to pass the linter checks?

Reply:
I think we've got a bit of all versions :smile:  
Personally I don't really mind the format, I think the order and grouping relating things matters more.